src/items/item.hpp

```diff
@@ -141,6 +141,24 @@ class ItemProperties {
 		return getCorpseOwner() == static_cast<uint32_t>(std::numeric_limits<int32_t>::max());
 	}
 
+	void setShader(const std::string &shaderName) {
```

Comment:
Maybe here can be "string_view"?

Reply:
it's a newbie question, but how do I use string_view without changing 
msg.addstring, setCustomAttribute, pushString. are previously defined as string.


![image](https://github.com/user-attachments/assets/38e58ced-6b42-4458-84c7-abe212797bd1)


![image](https://github.com/user-attachments/assets/bfe9671f-c39c-4d6c-8346-a107f3d3d15e)
![image](https://github.com/user-attachments/assets/155b510d-d2c7-45a0-a6ab-5a4dce4af8dc)
![image](https://github.com/user-attachments/assets/03384aa5-bba0-4e58-8be3-170f25f66c19)

Reply:
> it's a newbie question, but how do I use string_view without changing msg.addstring, setCustomAttribute, pushString.( are previously defined as string.)

In this case, it's preferable to keep it as a const reference. Using std::string_view would require creating a std::string with std::string(stringView), which would result in a similar overhead to passing the string by const reference.